boards/arm/mppt_1210_hus/mppt_1210_hus.dts

```diff
@@ -23,7 +23,7 @@
 	};
 
 	chosen {
-		zephyr,console = &usart1;
+		zephyr,console = &usart2;
```

Comment:
That's quite a significant change, as all other boards currently do not use the UEXT as the default console, but the other (debug) serial. So I'd not like to change that right now, at least not only for a single board and hidden in a commit with some other changes.